source/common/http/conn_manager_impl.cc

```diff
@@ -1783,6 +1790,7 @@ void ConnectionManagerImpl::ActiveStream::recreateStream(
   connection_manager_.doEndStream(*this, /*check_for_deferred_close*/ false);
 
   RequestDecoder& new_stream = connection_manager_.newStream(*response_encoder, true);
+  response_encoder->setRequestDecoder(new_stream);
```

Comment:
where was this before?

Reply:
This is new, we didn't previously set the new request decoder. Perhaps related to the comment in 1773-1775 (and perhaps I should change that comment).